expression/builtin_time.go

```diff
@@ -1770,6 +1770,72 @@ func getTimeZone(ctx context.Context) *time.Location {
 	return ret
 }
 
+// isDuration returns a boolean indicating whether the str is the format of duration.
```

Comment:
1. s/is the format/matches the format
2. add comment for the format that str should match.